docs/src/quill/getting-started.md

```diff
@@ -0,0 +1,70 @@
+# Getting started
+
+## Install Rust
+
+First you should install Rust, you can follow the guide below or visit [rustup.rs](https://rustup.rs/) directly.
+
+### To install Rust, if you are running Unix,
+run the following in your terminal, then follow the onscreen instructions.
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+### If you are running Windows 64-bit,
+download and run \
+[rustup-init.exe](https://win.rustup.rs/x86_64) \
+then follow the onscreen instructions.
+
+### If you are running Windows 32-bit,
+download and run \
+[rustup-init.exe](https://win.rustup.rs/i686) \
+then follow the onscreen instructions. 
+
+## Install Quill
+Second you are should install the Quill utility.
+
+To install Quill utility run the following in your terminal.
+```sh
+cargo install cargo-quill
+```
+
+## Creating a new plugin
+To create a new plugin in an exisiting directory
+```sh
+cargo quill init [path]
+```
+
+This command will create a new Cargo cdylib in the current directory.
+Give a path as an argument to create in the given directory.
+
+It creates will populate the directory with the following files.
+```
+src/
+└── lib.rs
+.gitignore
+cargo.toml
+```
+
+## Recomendations
+We also recomend that you install the following
+### Rust analyzer
```

Comment:
Also suggest IntelliJ Rust as an alternative.

Reply:
I have not used IntelliJ Rust before, maybe you or someone else who have used it could write that part?

Reply:
I've used it before, but in my experience the CLion version is better (still made by Jetbrains tho) since it has debugger support. I switched to vscode because the IntelliJ version was slower in my opinion.